docs/notifications.md

```diff
@@ -0,0 +1,41 @@
+# Notification Development Instructions
+
+The purpose of this document is to provide ZAP developers with simple instructions on how to add a notification.
+
+## Back End
+
+### Schema
+
+Within zap-schema.sql there is a table SESSION_NOTICE which is queried from the front end. SESSION_NOTICE has the following table entries:
+
+![session_notice](schema_notification.png)
+
+**SESSION_REF:** This entry is a key identifier for the current session. It is important because any warning or error that prompts a notification is only relevant to one session.
+
+**NOTICE_TYPE:** This entry is used as a label for the notification. For example, for a file upgrade the type would be "UPGRADE".
+
+**NOTICE_MESSAGE:** This entry is for the notification message.
+
+**NOTICE_SEVERITY:** This entry is for determining the level of severity of the notification. For example, an error that causes your project to completely fail would be higher severity while an issue that will not break your project but could cause confusion would be lower severity etc.
+
+**NOTICE_ORDER:** This entry is for organizing the entries in the "Notifications.Vue" page. **TODO:** higher priority items will have a higher order and therefore presented first or maybe with a more serious color representation like red.
+
+**DISPLAY:** This entry is for identifying a notification which would require a "pop up" rather than simply being shown on the Notifications page.
```

Comment:
Is this a boolean or integer?
If display is enabled for a notification then it will show as pop up on top of the zap UI apart from being shown in the notifications page.